tests/verify_collection.py

```diff
@@ -195,6 +253,13 @@ def get_bounding_box(granule_umm_json):
         south = bounding_box.get('SouthBoundingCoordinate')
         west = bounding_box.get('WestBoundingCoordinate')
         east = bounding_box.get('EastBoundingCoordinate')
+    
+    except ValueError:
```

Comment:
is there a better way to identify MLS collections for a global bounding box ?

Reply:
> is there a better way to identify MLS collections for a global bounding box ?

Looks like that code block wasn't needed after merging